### ISSUE TITLE
Catalog Data - catalog-data.sql

### ISSUE THREAD

**Issue by antciulla** [SoapboxRaceWorld/soapbox-race-core#80]
Why are we storing all this parsed XML in the SQL database rather then as plain XML files? Seems like it would be a waste of resources quarrying the SQL database rather then looking up the xml file on the server.

**Comment by LeoCodes21** [comment]
This was a design decision we made a long time ago - if memory serves, it was the way we did it in the original emulator, so it came along for the ride when we started developing this version.

A future release will introduce in-process caching for this table. Some other tables are cached at the moment, but the code dealing with this one hasn't received the caching treatment yet.

Hope this answers your question!

**Comment by antciulla** [comment]
Yes it does, I appreciate it. I'm trying to start familiarizing myself with the code so I can start contributing to the project, and this one one of the things that puzzled me.
